### ISSUE TITLE
Extension Error

### ISSUE THREAD

**Issue by t-vieira** [eusonlito/Password-Manager-Chrome#3]
Hello,

The system is working perfectly, but I installed the extension and when I enter the URL and API Key it gives the error:

`Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.`
`background.js:0 (função anônima)`

What could be wrong?

thanks

**Comment by eusonlito** [comment]
Do you have the extension installed from web chrome store? https://chrome.google.com/webstore/detail/eusonlito-password-manage/mkbgfbjaoibojobjpimpkaofckkgknhi

**Comment by t-vieira** [comment]
Yes, installed from Chrome web store, and cloning a repository gives the same error.

**Comment by eusonlito** [comment]
The extension fails on all pages? Some Google pages do not allow to open extensions.

**Comment by t-vieira** [comment]
The extension fails on all pages, it doesn't even show the list of passwords I have on the system, it just informs:

`No results, but you can add them through [WEB PANEL](https://pass.domain.com/).`

**Comment by eusonlito** [comment]
Some certificate problem with your API endpoint?

API Endpoint should be `https://XXXX.domain.com`

**Comment by t-vieira** [comment]
No, Let's Encrypt installed and working normally.
